### PR TITLE
Update docs jwt generator ui

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Update docs jwt generator ui

## What is the current behavior?

CleanShot 2023-01-17 at 21 13 56@2x

## What is the new behavior?

CleanShot 2023-01-17 at 21 13 30@2x

### CODE REVIEW

File: apps/docs/components/JwtGenerator.js [hunk 1]
```diff
@@ -57,19 +56,16 @@ export default function JwtGenerator({}) {
...
-        </select>
+        <Select id="service" onChange={handleKeySelection}>
+          <Select.Option value="anon">ANON_KEY</Select.Option>
+          <Select.Option value="service">SERVICE_KEY</Select.Option>
+        </Select>
       </div>
 
       <div className="grid mb-8">
         <label for="token">Payload:</label>
         <textarea
```
Comment (alaister) [inline apps/docs/components/JwtGenerator.js:67]: Will `<Input.TextArea />` work here instead of the custom input?

**Review by alaister — CHANGES_REQUESTED** [review]
Looks good! Just one question regarding the textarea:

File: apps/docs/components/JwtGenerator.js (hunk 1 above)
Comment (Isaiah-Hamilton) [reply apps/docs/components/JwtGenerator.js:67]: I haven't tried but let me check

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**